ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityActivityBlock.ts

```diff
@@ -0,0 +1,21 @@
+import { ActivityBlock } from './activityBlock';
+import { BlockType } from './blockType';
+import { ActivityInstance } from 'ddp-sdk';
```

Comment:
Importing library **_might_** be OK, but safer/more consistent to do a relative part import like the others.

Reply:
My IDE decided it instead of me 🙂
Fixed in d076b6949bc6943d9b7bd86d671b35b5e37ee643.

Reply:
I have same problem with IDE (I use IntelliJ). I think you are using WebStorm which is pretty much the same thing.
If you can figure out how to stop the IDE from doing these incorrect imports please let me know!